client/js/widgets/label.js

```diff
@@ -1,5 +1,37 @@
 import { Widget } from './widget.js';
 
+export async function unsetHTML(w) {
+  document.getElementById(w.id).innerHTML = null;
+	w.input = document.createElement('textarea');
+	 
+	w.addDefaults({
+    height: 20,
+    movable: false,
+    layer: -2,
+    typeClasses: 'widget label',
+
+    text: '',
+    editable: false,
+    twoRowBottomAlign: false
+    });
+  w.domElement.appendChild(w.input);
```

Comment:
This and the next line should be all you need. (And the first with `innerHTML=null`.)

Reply:
deleted